matrix-sdk/src/main/java/org/matrix/androidsdk/data/Room.java

```diff
@@ -3028,16 +3029,33 @@ private void sendTextMessage(String text, String HTMLFormattedText, String forma
      * @param listener           the event creation listener
      */
     public void sendMediaMessage(final RoomMediaMessage roomMediaMessage, final int maxThumbnailWidth, final int maxThumbnailHeight, final RoomMediaMessage.EventCreationListener listener) {
-        initRoomDataItemsSender();
+        initRoomMediaMessagesSender();
 
         roomMediaMessage.setThumnailSize(new Pair<>(maxThumbnailWidth, maxThumbnailHeight));
         roomMediaMessage.setEventCreationListener(listener);
 
         mRoomMediaMessagesSender.send(roomMediaMessage);
     }
 
+    /**
+     * Send an arbitrary event.
+     *
+     * @param event
+     * @param msgType
```

Comment:
In matrix wording, it should be `eventType`. `msgType` is a field in the event content when the event type is `m.room.message`.

Reply:
Done